modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -855,6 +859,18 @@ There are two static IPs that must be whitelisted to access the mule-manager hos
 |*EU*|*mule-manager.eu1.anypoint.mulesoft.com* |18.194.245.32
 |===
 
+[NOTE]
```

Comment:
Please change to:

The following DNS endpoints do not have static IP addresses: